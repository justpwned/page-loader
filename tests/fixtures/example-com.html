<!doctype html>
<html>
    <head>
        <title>Test</title>
        <link rel="stylesheet" media="all" href="/assets/styles.css">
    </head>
    <body>
        <div>
            <h1>Example Domain</h1>
            <p>This domain is for use in illustrative examples in documents.</p>
            <p><a href="https://www.iana.org/domains/example">More information...</a></p>

            <img src="/assets/test_image.jpg" alt="Test image">
            <img src="https://images.unsplash.com/photo-1636453966235-a5e76101975c">
        </div>
        <script src="https://js.stripe.com/v3/"></script>
    </body>
</html>
