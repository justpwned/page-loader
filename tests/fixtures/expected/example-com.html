<!DOCTYPE html>
<html>
 <body>
  <div>
   <h1>
    Example Domain
   </h1>
   <p>
    This domain is for use in illustrative examples in documents. You may use this
    domain in literature without prior coordination or asking for permission.
   </p>
   <p>
    <a href="https://www.iana.org/domains/example">
     More information...
    </a>
   </p>
   <img alt="Test image" src="example-com_files/example-com-assets-test-image1.jpg"/>
  </div>
 </body>
</html>
